test/cli.js

```diff
@@ -368,3 +371,17 @@ test('prefers local version of ava', function (t) {
 	t.ok(debugSpy.calledWith('Using local install of AVA'));
 	t.end();
 });
+
+test('use current working directory if `package.json` is not found', function (t) {
+	var dir = uniqueTempDir({create: true});
+	var testFilePath = path.join(dir, 'test.js');
+	var cliPath = require.resolve('../cli.js');
+	var avaPath = require.resolve('../index.js');
+
+	fs.writeFileSync(testFilePath, 'import test from \'' + avaPath + '\';\ntest(t => { t.pass(); });');
+
+	execa(process.execPath, [cliPath], {cwd: dir}).then(() => {
+		t.pass();
+		t.end();
+	});
```

Comment:
I think you should just return the promise. You don't test the output of the command, so no need for manual then/catch. Return a promise and test will fail, if the promise rejects.